esmvalcore/cmor/_fixes/shared.py

```diff
@@ -8,6 +9,170 @@
 logger = logging.getLogger(__name__)
 
 
+class AtmosphereSigmaFactory(iris.aux_factory.AuxCoordFactory):
```

Comment:
again, this class would benefit from full documentation and even migration into iris?

Reply:
To be honest, I just copy-pasted a similar class from `iris` and adapted it a little bit. If this definition makes scientifically sense, then this would definitely make way more sense in `iris`, but I don't have time for that right now, sorry :disappointed: